README.md

```diff
@@ -1 +1,53 @@
 # sec-directory-server
+
+This is the administration interface for the SEC Directory system. It's built on Django, utilizing docker to manage the build process and dependencies.
+
+## App architecture
+
+This application's primary purpose is to push directory data into a search index on Algolia. The main source for that data is the existing SEAS directory feed used for the www.seas.harvard.edu website; other people and places can be manually entered through the Django admin interface. Those three data sources are then compiled into a single "Rollup" data model, which is added to the Algolia index.
+
+A separate [sec-directory-client][client] project hosts the applications public-facing code, with connects directly to Algolia to search and display results from the index.
+
+## Django Tasks
+
+In addition to serving the administration interface, the Django app run some recurring tasks:
+
+- Importing and parsing the directory feed
+- Pushing the Data to the Algolia index
+
+**More details on those tasks and how they'll run in production, to come...**
+
+## Docker Setup
+
+### Development
+
+For development, there is a `docker-compose.yml` file in the root of the project. You'll need to copy the `template.env.dev` file to `.env` and fill in the appropriate values for the database connection and the Algolia application and index.
+
+For Algolia, you'll need to provide the `ALGOLIA_APP_ID` that owns the index defined in `ALGOLIA_INDEX`, and provide an `ALGOLIA_API_KEY` that has permission to `addObject`, `deleteObject` and `deleteIndex` for that same index. For security reasons, it's best [to create a "Secured API key"][api-key] scoped to the index.
+
+With the `.env` file in place, you can run `docker-compose up` to bring up the app and database containers. From there, you can access the Django admin interface in the browser at localhost:8000/admin and log in with the credentials defined in the `DJANGO_SUPERUSER_USERNAME` and `DJANGO_SUPERUSER_PASSWORD` variables.
+
+To access Django's CLI tool, you can run:
+
+```sh
+$docker-compose exec web python manage.py
+```
+
+That will list all of the available commands, but for development the most important ones will be:
+
+```sh
+# Run tests
+$ docker-compose exec web python manage.py test
```

Comment:
Ah right, so without this PR merged you'll have to pass the `--file docker-compose.dev.yml` argument, since the default is to use `docker-compose.yml`.

Both commands are doing the same thing, but I find the `docker-compose` version preferable because:

1. You don't have to pass `-it`, which I forget to do about half the time.
2. The name of the container is shorter and consistent (i.e. I have this repo cloned to subdirectory called `server` under `sec-directory`, so my container's called `server_web_1`)

Reply:
Ah I see, that makes sense, thanks!